src/components/navigation/header/config/insuranceComparison.ts

```diff
@@ -0,0 +1,63 @@
+import { NavigationLinkConfigProps } from './headerLinks';
+
+export const getAutoScoutInsuranceComparisonLinkConfig = (
+  userVisibilitySettings: NavigationLinkConfigProps['visibilitySettings']['userType'],
+): NavigationLinkConfigProps => ({
+  translationKey: 'header.searchMenu.insuranceComparison',
+  link: {
+    de: 'https://www.financescout24.ch/de/lp/autoversicherung-finden?utm_source=autoscout24.ch&utm_medium=web&utm_campaign=subnavigation_car_',
+    en: 'https://www.financescout24.ch/de/lp/autoversicherung-finden?utm_source=autoscout24.ch&utm_medium=web&utm_campaign=subnavigation_car_',
+    fr: 'https://www.financescout24.ch/fr/lp/trouver-assurance-auto?utm_source=autoscout24.ch&utm_medium=web&utm_campaign=subnavigation_car_',
+    it: 'https://www.financescout24.ch/it/lp/trova-assicurazione-auto?utm_source=autoscout24.ch&utm_medium=web&utm_campaign=subnavigation_car_',
```

Comment:
those are the same links visible in both user menu and top-level header for private